src/Encoding/ChainedFormatter.php

```diff
@@ -20,6 +20,11 @@ public static function default(): self
         return new self(new UnifyAudience(), new MicrosecondBasedDateConversion());
     }
 
+    public static function defaultWithUnixTimestampDates(): self
```

Comment:
Perhaps we could drop the "default" prefix? Using it wasn't my brightest moment (sleep deprivation and all, you know).